cmd/plan/pl001/command.go

```diff
@@ -46,17 +46,31 @@ func New(config Config) (*cobra.Command, error) {
 func mustLong() string {
 	var d time.Duration
 	for _, s := range Plan {
-		d += s.Backoff.Wait()
+		d += s.Backoff.Wait() + s.CoolDown
 	}
 
 	s := "Test plan pl001 launches a basic Tenant Cluster and verifies basic criteria like\n"
 	s += "the correct number of nodes and pods are running. Plan execution might take up\n"
 	s += "to " + d.String() + ".\n\n"
 
-	t := [][]string{{"ACTION", "RETRY", "WAIT"}}
+	t := [][]string{{"ACTION", "RETRY", "WAIT", "COOLDOWN"}}
```

Comment:
The whole table magic is repeated in the commands. I would refactor this separately. I just wanted to proof this is working and works for what we want to achieve.